stellarphot/settings/custom_widgets.py

```diff
@@ -83,19 +98,29 @@ def __init__(self, item_type_name, *arg, _testing_path=None, **kwargs):
             description=f"Delete this {self._display_name}",
         )
 
+        # Put almost everything into a VBox, which we can perhapds wrap in an
```

Comment:
misspelling of "perhaps" in comments... and I assume you are not using 'accordion' widget, so maybe drop mention of that....

Reply:
Fixewd